lib/instrumentation/@prisma/client.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2023 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const connection = Symbol('nrPrismaConnection')
+const url = require('url')
+const logger = require('../../logger').child({ component: 'prisma' })
+const parseSql = require('../../db/query-parsers/sql')
+
+/**
+ * Parses a connection string. Most database engines in prisma are SQL and all
+ * have similar engine strings.
+ *
+ * **Note**: This will not parse mongodb or sql server, instead will log a warning
```

Comment:
Hm, still kinda surprised to hear that. Reading the mongo docs, it uses standard URI for its connexion strings.

Reply:
you're right. not sure why I thought that